en/clubs/naringslivsgruppen/body.md

```diff
@@ -0,0 +1,13 @@
+# The Business Relations Group
+
+The Business Relations Group (NLG) at the computer science and engineering chapter  works to improve the relationship between the industry and computer science students. We make sure to create valuable contacts through interesting and engaging events, while also raising money for the chapter!
+
+If you are a company and want to get in touch with computer scientists, click [here](https://datasektionen.se/en/industry) to see what we offer!
+
+## What do we do?
+
+One of our most common events is lunch lectures. We invite a company to give a presentation during lunch (12.15-13.00) while food and drinks are served. Here, companies usually talk about themselves and bring up relevant jobs, projects or trainee programs that are interesting to the students.
+
+We also arrange company pubs, which means that a company participates and presents themselves at one of our Wednesday pubs. You can then mingle with the company in a more relaxed environment.
+
+We also take care of posting ads on discord, facebook and instagram from companies. It could be a job ad or an ad about other projects or programs the company is running that could be relevant and fun for computer science students.
```

Comment:
Most things look correct, but I recommend running the text through Grammarly or something similar.
It will fix the mistakes that are easy to make. Such as writing `facebook` instead of `Facebook`.

This is very nitpicky, but I think just making sure it doesn't contain mistakes is important for our business pages!